anchor/client/Cargo.toml

```diff
@@ -24,6 +24,7 @@ http_api = { workspace = true }
 http_metrics = { workspace = true }
 hyper = { workspace = true }
 keygen = { workspace = true }
+keysplit = { workspace = true }
```

Comment:
Do we need to add keysplit?

Reply:
My recommendation:  make docgen a separate workspace binary (e.g. tools/docgen) rather than an anchor client subcommand. This keeps the shipping CLI lean, avoids linking extra deps (e.g., keysplit) into the client crate.

Sketch:
	•	tools/docgen/Cargo.toml (bin = docgen)
	•	Depends on the crates that define the clap trees (e.g. anchor-client, keygen, keysplit) via path = "../anchor/client" etc.
	•	Each of those crates exposes a small pub fn cli() -> clap::Command (or impl CommandFactory) so docgen can introspect without booting the whole app.